wiki/Installation/Linux/en.md

```diff
@@ -0,0 +1,497 @@
+
+<!--TODO: add how to disable compositing without lutris -->
+
+### How to add osu! to Lutris
+
+1. Install Lutris trough its [download page](https://lutris.net/downloads)
+2. If you have never used Lutris before run this command
+
+```
+cd ~/.local/share/lutris/runners/ && mkdir wine
+```
+
+This will create your wine folder.
+
+3. Copy the Wine version you downloaded beforehand here
+
+```
+sudo cp -r /opt/wine-osu/ ~/.local/share/lutris/runners/wine
+```
+
+4. Open Lutris
+5. Click on the + button on the top left of your Lutris window and write osu! in the name. For the runner, choose Wine.
+6. Go to the `Game options` tab and set it up like this:
+
+Executable: `~/.local/share/osu-wine/OSU/osu!.exe`\
+Working directory: `~/.local/share/osu-wine/OSU`\
+Wine prefix: `~/.local/share/osu-wine/WINE.win32`
+
+7. Go to the `Runner options` tab and here select "wine-osu" as the wine version.
+8. Disable desktop effects in `System options` tab (see [Input_lag/Compositing](#Input_lag/Compositing))
+
+### osu! installer can't download files/XComposite error etc
+
+All these errors come from missing dependencies, make sure to install them all (see see [Installing Wine and its dependencies](#Installing_Wine_and_its_dependencies)).
+
+### No sound when opening osu!
+
+You probably only need to reboot or reload PipeWire using these commands:
+
+```
+sudo systemctl --user restart pipewire pipewire-pulse
+sudo systemctl --user daemon-reload
+```
+
+## Additional Informations
+
+### - Optimize your performance: Gamemode
+
+Gamemode is a tool which might improve your gaming depending on your system. To install it and use it, follow these instructions by spunout enjoyer:
+
+1. Install Gamemode
+
+### Debian based
+
+```
+sudo apt install gamemode
+```
+
+### Arch based
+
+```
+sudo pacman -S gamemode lib32-gamemode
+```
+
+### Fedora
+
+```
+sudo dnf install gamemode
+```
+
+2. Use Gamemode by this command; or
+
+```
+gamemoderun osu-wine
+```
+
+4. Enable Gamemode by default in Lutris by following these steps
+
+![](img/lutris-enable-gamemode.gif)
+
+### - Even lower latency: gonX's PipeWire settings
+
+If after installing osu! your latency sounds strange and you want an even lower one, we'll apply gonX's PipeWire "patch"
+
+1. To prevent any missing files, run this command:
+
+```
+mkdir -p ~/.config/pipewire && cp -rv /usr/share/pipewire/* ~/.config/pipewire/
+```
+
+2. Edit the `pipewire-pulse.conf` file in `~/.config/pipewire/`
+
+```
+sudo nano ~/.config/pipewire/pipewire-pulse.conf
+```
+
+3. Here, you might want to push your latency as far as possible, starting from 256, try to find the closest value to 32 which makes your sound stable/not cracking. It depends on your PC. For example:
+
+```
+context.modules = {
+pulse.min.req = 32/48000 # 0.67ms
+pulse.min.quantum = 32/48000 # 0.67ms
+}
+```
+
+4. Save with Ctrl+O and exit with Ctrl+X.
+5. Reboot your PC.
+6. Enable compatibility mode on osu!.
+
+### - PP Counter for OBS: gosumemory
```

Comment:
This kind of stuff most *definitely* cannot be on the official wiki.

Reply:
if you are talking about those additional info, it can just be removed

Reply:
No applications, links to executables, etc. can be in this guide.

I would propose this is just a stub leading to off-site guides to getting stuff working. I don't believe it should be present in the official wiki. Officially we do not support linux, and don't intend to offer support that isn't via osu!(lazer).

In fact I'd argue that this guide should be about osu!(lazer) instead.

Reply:
how about, move this page content to my gist (or maybe forum post) and then link that to this page.
about osu!lazer

Reply:
As long as it doesn't link to custom osu/wine packages. It should be instructions on how to make things work with no external binaries.